AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -1251,6 +1251,7 @@ benesse.jp##.bnr-bottom-art
 benesse.jp##.bnr-overlay-art
 benesse.jp##.bnr_flow-under-h2
 benesse.jp##.sh-appbnr
+cat.benesse.ne.jp,dog.benesse.ne.jp##.app_download_area
```

Comment:
For this particular rule and the one in Social, you can use `benesse.ne.jp` instead of `cat.benesse.ne.jp,dog.benesse.ne.jp` as false positive is very unlikely. But leave ad rules as currently are.

Reply:
OK, I will omit the domain.